bin/mail/2017-08-30-pd-pre-workshop-survey-correction/generate-list

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+require_relative '../../../dashboard/config/environment'
+
+# The original mail with the erroneous link was sent as the 10-day reminder email on the morning of 2017-08-30
+# Make sure we're running an equivalent query of 10 days before 8-30
+days_before = (Date.parse('2017-08-30') + 10.days - Date.today).to_i
+raise 'Must be run before 2017-08-30' if days_before < 1
```

Comment:
The error message does not seem to reflect when the `days_before < 1` condition happens?

Reply:
Oh right, good point - it should be before 10 days from today not before today :(

I already sent the emails so we won't need to run this again. Think it's worth updating the text anyway? I can prep a followup.

Reply:
Probably not worth it. This comment thread should suffice, IMO.